net-libs/libwebsockets/metadata.xml

```diff
@@ -19,7 +19,6 @@
 	</maintainer>
 	<use>
 		<flag name="access-log">Support generating Apache-compatible access logs</flag>
-		<flag name="cgi">Include CGI (spawn process with network-connected stdin/out/err) APIs</flag>
```

Comment:
The default value is 

>Add CGI script support

Would you say the current metadata flag has a different meaning?

Reply:
Global description is correct, so use that.

Reply:
Yup, thank you